src/sentry/static/sentry/app/components/group/pluginActions.tsx

```diff
@@ -112,7 +129,7 @@ class PluginActions extends React.Component {
           enforceFocus={false}
         >
           <Modal.Header closeButton>
-            <Modal.Title>{`${plugin.name || plugin.title} Issue`}</Modal.Title>
```

Comment:
Are we sure that `name` always exists and is not an empty string?

Reply:
@billyvg It should be but I was able to keep this by fixing the type